torch/quantization/_generated/QAddGen.py

```diff
@@ -0,0 +1,33 @@
+
+# This file is generated using `torch.nn.quantization.make_module`
+# and saved as `QAddGen.py`.
+
+import torch
+
+r"""AddGen wraps the torch.add function."""
+class AddGen(torch.nn.Module):
+    def __init__(self, **kwargs):
+        super(AddGen, self).__init__()
+
+    def forward(self, *args):
+        return self.torch.add(*args)
+
+
+r"""QAddGen wraps the torch.ops.quantized.add function."""
+class QAddGen(torch.nn.Module):
+    def __init__(self, **kwargs):
+        super(QAddGen, self).__init__()
+
+        self.scale = kwargs.get('scale', 1.0)
```

Comment:
why kwags - can't they be regular args?

Reply:
Because during the generation I am reusing the `init` part, and in non-quantized modules we don't have the scale and zero_point, they might or might not exist, depending on what is being generated.